app/js/auth/index.js

```diff
@@ -7,7 +7,8 @@ import cookie from 'js-cookie';
 
 class Auth {
   andelaLoginUrl() {
-    return `${cookie.get(CVar.auth_url)}?redirect_url=http://${
+    const auth_url = cookie.get(CVar.auth_url) || 'http://api-staging.andela.com/login';
```

Comment:
Why not have the URL `http://api-staging.andela.com/login` in the config file? @cyrielo